$NetBSD: patch-Makefile,v 1.1.1.1 2011/05/16 13:52:56 wiz Exp $

Install into PREFIX, and remove PYTHON override (set from pkgsrc Makefile).

--- Makefile.orig	2011-03-20 07:40:42.000000000 +0000
+++ Makefile
@@ -15,13 +15,12 @@ bup: lib/bup/_version.py lib/bup/_helper
 Documentation/all: bup
 
 INSTALL=install
-PYTHON=python
-MANDIR=$(DESTDIR)/usr/share/man
-DOCDIR=$(DESTDIR)/usr/share/doc/bup
-BINDIR=$(DESTDIR)/usr/bin
-LIBDIR=$(DESTDIR)/usr/lib/bup
+MANDIR=$(DESTDIR)$(PREFIX)/share/man
+DOCDIR=$(DESTDIR)$(PREFIX)/share/doc/bup
+BINDIR=$(DESTDIR)$(PREFIX)/bin
+LIBDIR=$(DESTDIR)$(PREFIX)/lib/bup
 install: all
-	$(INSTALL) -d $(MANDIR)/man1 $(DOCDIR) $(BINDIR) \
+	$(INSTALL) -d $(BINDIR) \
 		$(LIBDIR)/bup $(LIBDIR)/cmd $(LIBDIR)/tornado \
 		$(LIBDIR)/web $(LIBDIR)/web/static
 	[ ! -e Documentation/.docs-available ] || \
